LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/trad-chinese-formal/css3-counter-styles-088-expected.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
-<html  lang="en" >
+<html lang="en">
 <head>
-<meta charset="utf-8"/>
+<meta charset="utf-8">
 <title>trad-chinese-formal, outside range</title>
-<link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
+<link rel="author" title="Mats Palmgren" href="mailto:mats@mozilla.com">
 <link rel='help' href='http://www.w3.org/TR/css-counter-styles-3/#complex-cjk'>
-<link rel="reviewer" title="Chris Lilley" href="mailto:chris@w3.org" />
-<meta name="assert" content="[Exploratory] list-style-type: trad-chinese-formal produces counter values outside its range without using the prescribed fallback style.">
-<style type='text/css'>
+<link rel='help' href='https://bugzilla.mozilla.org/show_bug.cgi?id=1738356'>
+<style>
```

Comment:
I'm guessing these are import changes? It seems weird to change the credits given the the test writer.